data.sql

```diff
@@ -4,3 +4,8 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon','11/15/2018',2,true,8);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu','01/07/2021',1,false,15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon','05/16/2017',5,true,11);
+
+-- Species
```

Comment:
## Missing Requirements
![Screenshot from 2022-08-30 13-46-21 (copy)](https://user-images.githubusercontent.com/93039359/187392745-df1b4254-e343-40a3-bef4-7b5c02c768a3.png)


- [ ] I guess your forgot to add these queries shown in the red boxes above :smile: Please add them to make your project ready for approval :+1:

Reply:
@HAMMAS-SALEEM hey , i added this queries in the screenshots [9 to 13] please check , there's no requirement to add them in the file , so i'm not supposed to add them in the file , thank you !